server/athenian/api/internal/miners/github/pull_request.py

```diff
@@ -1119,6 +1119,7 @@ async def fetch_jira():
                     ],
                 )
                 df[PullRequest.node_id.name] = df[PullRequest.node_id.name].astype(int)
+                df[_issue.project_id.name] = df[_issue.project_id.name].astype("S8")
```

Comment:
Let me debug this separately, we shouldn't need this, in theory. Can be another pandas glitch.

Reply:
https://github.com/athenianco/athenian-api/runs/8249382270?check_suite_focus=true#step:12:232
This is the failing test without

Reply:
OK, I will investigate after the merge.

Reply:
I see. This is a new dataframe without dtypes. All right.